tests/benchdnn/graph/graph.cpp

```diff
@@ -557,6 +544,19 @@ int doit(const prb_t *prb, res_t *res) {
         return FAIL;
     }
 
+    if (prb->expected_n_partition != 0) {
+        // If the expected partition num is specified by user with command line
```

Comment:
a little confusion in comments, **_1_** is default but not specified by cml

Reply:
Thanks. I will remove this comment.

Reply:
Maybe fuse two conditions:
```
if (prb->expected_n_partition > 0 && partitions.size() != prb->expected_n_partition) {
    ...
}
```